tutorials/model_exploration.md

```diff
@@ -0,0 +1,75 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Model Exploration
```

Comment:
I'm not sure this tutorial really tells you how to 'explore' a model. Maybe it should just be "Loading and Querying a Model"

Reply:
I'll retract this comment, the learning objectives make it obvious enough what this is going to show!

Reply:
Thanks. I generally try to keep the titles as short as possible.